entry.js

```diff
@@ -6,9 +6,12 @@
 require('babel-register')
 require('babel-polyfill')
 
+// The code forced me to overwrite native functions. I'm sorry
+global.Promise = require('bluebird')
```

Comment:
It's ok, we forgive you. Because bluebird is the best promise implentation I've seen so far :+1: